frontend/express/public/core/geo-countries/javascripts/countly.views.js

```diff
@@ -0,0 +1,291 @@
+                {"value": "n", "label": CV.i18n('common.table.new-users'), "trend": totals.n.trend, "number": countlyCommon.getShortNumber(totals.n.total || 0), "trendValue": totals.n.change}
+            ];
+        },
+        geoChartData: function() {
+            var geoChart = [];
+            var table = this.data.table || [];
+            var selectedProperty = this.$store.state.countlyCountry.selectedProperty || "t";
+
+            var title = "";
+            if (selectedProperty === "t") {
+                geoChart.push(["Country", CV.i18n('common.table.total-sessions'), {role: 'tooltip', p: {html: true}}]);
+                title = CV.i18n('common.table.total-sessions');
+            }
+            else if (selectedProperty === "u") {
+                geoChart.push(["Country", CV.i18n('common.table.total-users'), {role: 'tooltip', p: {html: true}}]);
+                title = CV.i18n('common.table.total-users');
+            }
+            else {
+                geoChart.push(["Country", CV.i18n('common.table.new-users'), {role: 'tooltip', p: {html: true}}]);
+                title = CV.i18n('common.table.new-users');
+            }
+            for (var k = 0; k < table.length; k++) {
+                var cc = "<img src='" + countlyGlobal.path + "/images/flags/" + table[k].code + ".png'/><p class='number'>" + table[k][selectedProperty] + "</p><p>" + title + "</p>";
+                geoChart.push([{"v": table[k].country, "f": table[k].countryTranslated}, table[k][selectedProperty], cc]);
+            }
+            return geoChart;
+        },
+        countriesData: function() {
+            var geoChart = {};
+            var table = this.data.table || [];
+            var selectedProperty = this.$store.state.countlyCountry.selectedProperty || "t";
+            for (var k = 0; k < table.length; k++) {
+                geoChart[table[k].country] = {"value": table[k][selectedProperty]};
+            }
+            return geoChart;
+        },
+        countryTable: function() {
+            this.data = this.data || {};
+            this.data.table = this.data.table || [];
+            for (var z = 0; z < this.data.table.length; z++) {
+                this.data.table[z].flag = "flag " + this.data.table[z].code;
+                this.data.table[z].styling = "margin-top:2px; background-image:url(" + countlyGlobal.path + "/images/flags/" + this.data.table[z].code + ".png);";
+            }
+            return this.data.table;
+        },
+        selectedProperty: {
+            set: function(value) {
+                this.$store.dispatch('countlyCountry/onSetSelectedProperty', value);
+            },
+            get: function() {
+                return this.$store.state.countlyCountry.selectedProperty;
+            },
+            dropdownsDisabled: function() {
```

Comment:
Probably a mistake. I think `dropdownsDisabled` is supposed to be a computed field while you define it in the selectedBrowser field.

I noticed this one of Pınar's PRs too. Is this coming from somewhere?